test/e2e/xtx/manager.go

```diff
@@ -0,0 +1,99 @@
+package xtx
```

Comment:
not sure about this package name? these are not "cross chain transactions". They are normal transactions. 

I'd suggest something simple: `send.Sender`

Reply:
Also please package comment